docs/modules/general/main.mdx

```diff
@@ -155,10 +158,34 @@ Additionally, a variant can also contain constants, which allows you to define t
 
 | Attribute | Description | Value |
 |---|---|---|
-| `variant` | The name of the variant to target. | <span className="badge badge--primary">Variant ID</span> |
+| `variant` | The map variant to target.<br />*Multiple variants can be targeted as long as it is separated with a comma (`,`).* | <span className="badge badge--primary">Variant ID</span> |
 | `has-variant` | Target all maps with a specified variant.<br />**Note:** This can be useful in a server's global XML file. | <span className="badge badge--primary">Variant ID</span> |
-
-**Note:** Multiple variants can be targeted as long as it is separated with a comma (`,`).
+| `min-server-version` | The minimum server version this variant can run on.<br />**Note:** PGM will attempt to detect the map version from its `level.dat` file. |
+| `max-server-version` | The maximum server version this variant can run on. |
+| `constant` | The name of the constant to check against.<br />*Constants must be defined before this conditional.* | <span className="badge badge--primary">Constant ID</span> |
+| `constant-value` | The constant value to check for.<br />*Multiple values can be checked as long as it is separated with a comma (`,`).* | <span className="badge badge--secondary">Constant Value</span>
```

Comment:
comma separated multiple checks is only a thing when using `constant-comparison="contains"`, i'd remove the note saying that tbh

Reply:
Note has been removed now 👍